core/signer/signer.go

```diff
@@ -122,7 +122,7 @@ func (s *Signer) StartSigning(ctx context.Context, minInterval, maxInterval time
 	last := time.Unix(0, mapRoot.GetTimestampNanos())
 	// Start issuing epochs:
 	clock := util.SystemTimeSource{}
-	tc := time.Tick(minInterval)
+	tc := time.NewTicker(minInterval).C
```

Comment:
Did one of the linter checks recommend this change? `time.Tick` is just a wrapper around `time.NewTicker().C`